src/types/onyx/Policy.ts

```diff
@@ -32,6 +32,10 @@ type Policy = {
     isFromFullPolicy?: boolean;
     lastModified?: string;
     customUnits?: Record<string, unknown>;
+
+    areChatRoomsEnabled: boolean;
+
+    isPolicyExpenseChatEnabled: boolean;
```

Comment:
Can we add comments to all of these types for consistency? Seems like we stopped halfway through. And some could be better explained e.g. what determines if these two booleans are true? Are we able to enable/disable these features on a policy specifically?